mxcubecore/HardwareObjects/abstract/AbstractActuator.py

```diff
@@ -175,10 +204,10 @@ def re_emit_values(self):
         super(AbstractActuator, self).re_emit_values()
 
     def force_emit_signals(self):
-        """Forces to emit all signals.
+        """
+        Forces to emit all signals.
```

Comment:
"Force" seems implied.

```suggestion
        Emit all signals.
```

Alternatively, in the next file, you've used the word "Reemit".